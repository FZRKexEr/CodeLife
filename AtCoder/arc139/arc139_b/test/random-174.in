100
47 14 16 8 11 5
37 18 8 7 7 20
61 9 10 5 5 7
26 1 20 15 5 20
48 18 4 6 8 14
67 10 6 11 1 19
47 5 17 4 20 18
16 10 16 9 9 5
66 6 16 6 12 13
4 11 9 17 16 1
81 20 16 4 1 4
43 18 18 17 1 7
22 3 10 11 14 4
26 2 4 4 12 13
42 17 2 10 12 2
1 2 11 16 1 10
10 20 19 10 7 4
30 9 18 7 3 5
79 4 5 8 15 6
11 2 19 18 8 13
54 17 12 2 18 6
19 3 8 5 11 5
40 6 4 15 20 13
65 17 5 13 16 2
47 12 7 18 16 17
100 6 12 19 12 16
37 2 4 8 16 12
71 2 16 13 1 3
68 19 8 4 19 20
61 20 8 4 14 8
3 17 15 12 20 7
27 14 10 10 10 5
34 14 14 15 14 6
15 9 15 12 12 17
6 13 13 20 11 7
21 20 6 9 16 1
64 5 3 18 9 19
57 3 18 4 3 2
89 20 18 6 18 8
8 4 18 17 6 19
95 4 18 7 18 5
9 10 15 1 13 13
90 18 5 1 4 7
23 4 2 12 1 5
31 2 15 20 7 19
29 7 18 3 14 18
40 15 17 11 15 4
29 5 18 17 8 13
41 2 9 8 9 17
28 3 4 20 19 10
15 1 19 2 8 15
85 5 18 5 18 5
79 16 18 18 10 10
92 16 16 6 1 4
19 6 19 10 12 10
10 20 7 3 2 7
95 7 16 3 10 4
72 13 13 8 1 3
55 11 19 4 16 10
19 9 6 15 17 8
29 16 12 4 11 12
16 7 11 18 5 16
64 14 11 5 1 6
9 13 1 1 3 19
99 3 16 12 5 5
64 17 19 16 14 2
49 7 8 4 6 7
9 11 3 16 7 16
38 1 9 19 3 5
77 12 10 19 9 7
38 5 10 7 3 18
70 4 5 12 9 10
59 14 9 8 16 20
48 4 4 4 10 4
100 16 2 8 6 8
38 11 20 19 1 17
74 1 6 4 9 10
31 1 17 5 4 8
15 17 8 2 5 5
95 14 20 4 11 20
89 2 2 8 8 18
36 12 20 15 19 8
20 14 4 11 8 13
95 19 18 4 6 20
99 13 19 4 20 11
73 1 13 4 19 19
70 16 17 20 17 9
54 18 3 4 9 1
93 19 20 11 9 3
66 7 12 15 7 9
19 11 1 6 2 18
92 6 3 20 3 14
73 17 8 17 16 17
47 17 12 11 1 20
33 4 11 10 16 15
63 13 9 10 20 5
66 1 5 14 14 20
45 8 16 1 17 19
70 17 7 7 2 6
8 20 17 11 10 6
