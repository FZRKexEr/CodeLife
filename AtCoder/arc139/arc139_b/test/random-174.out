29
21
47
50
58
36
88
5
64
68
8
121
30
86
32
16
100
29
105
58
36
21
130
26
101
204
116
28
89
46
36
45
102
17
120
25
145
9
126
12
50
9
20
22
58
44
74
46
94
70
30
71
166
77
10
16
43
61
60
39
27
46
15
9
35
118
42
48
26
74
42
140
88
48
52
98
128
64
19
95
360
57
43
30
71
127
76
18
155
60
50
85
135
33
45
35
274
45
22
88
